Assets/Scripts/PlayerController.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class PlayerController : MonoBehaviour {
+
+    //private Rigidbody playerRB to control player movement
+    private Rigidbody playerRB;
+
+    //publc float speed to modify the speed of the player
+    public float speed;
```

Comment:
Use [SerializeField] on a private/protected variable to make it editable in the editor.

Reply:
If that's the reason it's public, which I'm assuming it is.